chipsec/modules/common/memlock.py

```diff
@@ -45,14 +46,16 @@ class memlock(chipsec.module_common.BaseModule):
 
     def __init__(self):
         BaseModule.__init__(self)
+        self.res = ModuleResult.NOTAPPLICABLE
 
     def is_supported(self):
         # Workaround for Atom based processors.  Accessing this MSR on these systems
         # causes a GP fault and can't be caught in UEFI Shell.
-        if self.cs.get_chipset_id() in chipsec.chipset.CHIPSET_FAMILY_ATOM:
-            return False
-        return True
-        
+        if self.cs.get_chipset_id() not in chipsec.chipset.CHIPSET_FAMILY_ATOM:
```

Comment:
This could be reduced to a self.cs.is_atom() call.  Not required unless you really want to change it.

Reply:
I think the same can be done in multiple modules, going to merge this in and make a more general change